src/components/markdown/ProblemsList/ProblemsListItemDropdown.tsx

```diff
@@ -45,13 +44,13 @@ function ViewSolutionsContent({
     );
   }
   return (
-    <button
-      type="button"
+    <Link
       className="focus:outline-none block w-full text-left px-4 py-2 text-sm text-gray-700 dark:text-gray-300 hover:bg-gray-100 dark:hover:bg-gray-800 hover:text-gray-900"
-      onClick={onViewProblemSolutions}
+      to={`/problems/${problem.uniqueId}/user-solutions`}
+      target="_blank"
     >
```

Comment:
can you still use `<Link>` with target blank or do you just use `<a>` here?

Reply:
Both function the same, lmk if you want to change it to an anchor tag instead

Reply:
Maybe change it just to be safe? https://github.com/gatsbyjs/gatsby/issues/13825 (also on the preview deployment it doesn't seem to open in a new tab)